src/components/authentication/Login.jsx

```diff
@@ -24,16 +32,40 @@ const Login = () => {
   };
 
   // Handle the form submission
-  const handleSubmit = (event) => {
-    console.log({ username, password });
+  const handleSubmit = async (event) => {
     event.preventDefault();
+    try {
+      const response = await axios.post(`${BaseUrl}/auth/login/`, {
+        username: username.value,
+        password: password.value,
+      });
+      if (response.status === 200) {
+        const token = response.data.token;
+        const user = response.data.user;
+        let expiresIn = new Date();
+        expiresIn.setTime(expiresIn.getTime() + (60 * 1000));
+        Cookies.set('access_token', token, { path: '/', expires: 1, secure: true });
+        Cookies.set('user', JSON.stringify(user), { path: '/', expires: 1, secure: true });
+        setMessage({
+          type: 'success',
+          body: 'Log in successfully',
+        });
+        navigate('/');
+      } else {
+        alert(response.data.error);
+      }
+    } catch (error) {
+      console.error('Axios error:', error);
+      alert('An error occurred while making the request. Please try again.');
+    }
```

Comment:
Move this along with other api logic in a separate file like `api.js` and import them here. This will improve readability and separate out the axios calls from components.